examples/jenkins/pipeline/pipelinetemplate.json

```diff
@@ -19,6 +19,9 @@
         "creationTimestamp": null,
         "labels": {
           "name": "sample-pipeline"
+        },
+        "annotations": {
+          "downstream.openshift.io/deployments": "[{\"name\": \"frontend\", \"namespace\": \"\"}]"
```

Comment:
Did we pick a name? I've been using `downstream.openshift.io/deployments` for the time being.